aries/aries_vcx/src/handlers/out_of_band/receiver.rs

```diff
@@ -38,6 +41,33 @@ impl OutOfBandReceiver {
         }
     }
 
+    pub fn create_from_json_encoded_oob(oob_json: &str) -> VcxResult<Self> {
+        Ok(Self {
+            oob: extract_encoded_invitation_from_json_string(oob_json)?,
+        })
+    }
+
+    // This is here to help facilitate testing
```

Comment:
what's the point in keeping this method if it's only usage is in a test?

Reply:
I thought it was important to be able to validate whether we can handle padded and unpadded, but on reconsideration I should just make the test do that via the URL, not via an API we don't need. 
Thanks for helping me think through that lol, I'll update by the morning 👍